pyproject.toml

```diff
@@ -54,11 +54,10 @@ dependencies = [
     "lxml>=4.5.2",
     "PyRSS2Gen>=1.1",
     "blinker>=1.3",
-    "setuptools>=60.0.5",
```

Comment:
Can you mention this in `CHANGES.txt` in a new top-level section `New in master`?

Reply:
Sure, done.